### PR TITLE
new structure

### PR DESCRIPTION
**Pull request by 1over**

### CODE REVIEW

File: modules/__init__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from . import sasai
+from . import say
+from . import timer
+
+
+def get_handlers():
+    from functools import reduce
+    modules = [bomb_word, chat, infometr, sasai, say, timer]
+    handlers = [getattr(m, 'handlers', []) for m in modules]
+    return reduce(lambda x, y: x + y, handlers)
```
Comment (kramar42) [inline modules/__init__.py:15]: if i understand correctly, that's list flattening? maybe better do it with list comprehension, i.e.:
`
return [handler for m in modules for hander in getattr(m, 'handlers', [])]
`

**Review by kramar42 — COMMENTED** [review]
nice simplification, always happy to see more red than green

**Comment by 1over** [comment]
> nice simplification, always happy to see more red than green

Indeed, without music module we don't need all this logic to reload modules in runtime

File: modules/__init__.py (hunk 1 above)
Comment (1over) [reply modules/__init__.py:15]: good point, will fix

**Review by eluppol — APPROVED** [review]

**Pull request merged**